sjb/config/common/test_cases/origin_release_install_azure.yml

```diff
@@ -0,0 +1,194 @@
+---
+parent: 'common/test_cases/origin_release_with_ecosystem.yml'
```

Comment:
Can we run our job against openshift/origin#master today? Shouldn't we test release-3.9 instead (in which case this should be `common/test_cases/origin_release_with_ecosystem_39.yml`)?